### ISSUE TITLE
java.lang.UnsupportedOperationException thrown

### ISSUE THREAD

**Issue by guoci** [Nesvilab/FragPipe#243]
Using FragPipe 14 on Linux with workflow `open` on a single `.raw` file, I got the following error.
```
Crystal-C [Work dir: /home/ci/out]
java -Dbatmass.io.libs.thermo.dir="/home/ci/msfragger/target/ext/thermo" -Xmx113G -cp /home/ci/FragPipe/MSFragger-GUI/build/github-release/fragpipe/tools/original-crystalc-1.3.2.jar:/home/ci/FragPipe/MSFragger-GUI/build/github-release/fragpipe/tools/batmass-io-1.19.5.jar:/home/ci/FragPipe/MSFragger-GUI/build/github-release/fragpipe/tools/grppr-0.3.23.jar crystalc.Run /home/ci/out/crystalc-0-PRF_Q_2019_R_OHI_92_47583.pepXML.params /home/ci/out/PRF_Q_2019_R_OHI_92_47583.pepXML
Exception in thread "main" java.lang.UnsupportedOperationException: Batmass-IO binaries for Thermo support and/or Thermo native libraries not found found
	at com.dmtavt.batmass.io.thermo.ThermoGrpcServerProcess.<init>(ThermoGrpcServerProcess.java:70)
	at umich.ms.fileio.filetypes.thermo.ThermoRawFile.init(ThermoRawFile.java:202)
	at umich.ms.fileio.filetypes.thermo.ThermoRawFile.<init>(ThermoRawFile.java:161)
	at crystalc.p_ReadData.LoadRawFile(p_ReadData.java:166)
	at crystalc.Run.ProcessPepXML(Run.java:70)
	at crystalc.Run.main(Run.java:51)
Process 'Crystal-C' finished, exit code: 1
```
[log_2020-10-02_12-27-47.txt](https://github.com/Nesvilab/FragPipe/files/5319455/log_2020-10-02_12-27-47.txt)

**Comment by fcyu** [comment]
Can MSFragger search the raw file?

**Comment by guoci** [comment]
@fcyu yes. The log is attached.

**Comment by fcyu** [comment]
@hayse1 Told me the same issue yesterday but I cannot reproduce it in Windows. `-Dbatmass.io.libs.thermo.dir="/home/ci/msfragger/target/ext/thermo"` also looks correct. So, I don't know what the issue is.

**Comment by guoci** [comment]
Fixed in commit 6b0434b0282a2cbb595bf23802f0915f4464294a.
